examples/plot_tuh_eeg_corpus.py

```diff
@@ -64,14 +116,15 @@
     alpha=.5,
 )
 ax.legend(genders)
+ax.set_xlabel('Age [years]')
+ax.set_xlabel('Count')
```

Comment:
```suggestion
ax.set_ylabel('Count')
```

Mistake in axis label...